native/src/routes/Pois.tsx

```diff
@@ -59,29 +60,47 @@ type PoisProps = {
   navigation: NavigationProps<PoisRouteType>
 }
 
+const RESTORE_TIMEOUT = 100
+
 const Pois = ({ pois, language, cityModel, route, navigation }: PoisProps): ReactElement => {
   const { coordinates, requestAndDetermineLocation } = useUserLocation(true)
   const { slug } = route.params
   const [sheetSnapPointIndex, setSheetSnapPointIndex] = useState<number>(1)
   const [followUserLocation, setFollowUserLocation] = useState<boolean>(false)
+  const [listScrollPosition, setListScrollPosition] = useState<number>(0)
   const deviceHeight = useWindowDimensions().height
   const features = prepareFeatureLocations(pois, coordinates)
   const selectedFeature = slug ? features.find(it => it.properties.slug === slug) : null
   const poi = pois.find(it => it.slug === slug)
   const { t } = useTranslation('pois')
   const theme = useTheme()
   const cameraRef = useRef<MapboxGL.Camera | null>(null)
-
+  const scrollRef = useRef<BottomSheetScrollViewMethods>(null)
+
+  const scrollTo = (position: number) => {
+    setTimeout(() => {
+      if (scrollRef.current) {
+        scrollRef.current.scrollTo({
+          y: position,
+          animated: false,
+        })
+      }
+    }, RESTORE_TIMEOUT)
```

Comment:
couldn't get it work without timeout, event when i trigger it in the useEffect after `setCamera` something seems not to be ready